packages/platform-sdk-profiles/src/wallets/wallet.models.ts

```diff
@@ -42,6 +42,7 @@ export enum WalletData {
 	Delegates = "DELEGATES",
 	ExchangeCurrency = "EXCHANGE_CURRENCY",
 	ExchangeRate = "EXCHANGE_RATE",
+	ExchangeRates = "HISTORICAL_EXCHANGE_RATE",
```

Comment:
```suggestion
	ExchangeRates = "EXCHANGE_RATES",
```

The value should always be the `upper-case snake_case` of the key.